### ISSUE TITLE
like_superlike: like_get_urlpic error

### ISSUE THREAD

**Issue by focamonca** [social-manager-tools/instagram-bot.js#86]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.9.0
- **Node Version:** lastest
- **Operating System:** Raspbian 9
- **Browser:** chromium

### Expected Behavior

Please describe the program's expected behavior.

```
The bot should run smoothly
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
[INFO] like_superlike: try open userpage
[DEBUG] like_superlike: Error: waiting for selector "main article:nth-child(1) header:nth-child(1) div:nth-child(2) a:nth-child(1)" failed: timeout 30000ms exceeded
[INFO] like_superlike: photo not found, retry next...
[INFO] like_superlike: like_get_urlpic
[DEBUG] like_superlike: array photos 
[INFO] like_superlike: current photo url undefined
[WARNING] like_superlike: check if current hashtag have photos, you write it good in config.js? Bot go to next hashtag.
[ERROR] like_superlike: like_get_urlpic error Error: Protocol error (Page.navigate): Invalid parameters url: string value expected
[INFO] like_superlike: try open userpage
[DEBUG] like_superlike: Error: waiting for selector "main article:nth-child(1) header:nth-child(1) div:nth-child(2) a:nth-child(1)" failed: timeout 30000ms exceeded
[INFO] like_superlike: photo not found, retry next...
[INFO] like_superlike: like_get_urlpic
[DEBUG] like_superlike: array photos 
[INFO] like_superlike: current photo url undefined
[WARNING] like_superlike: check if current hashtag have photos, you write it good in config.js? Bot go to next hashtag.
[ERROR] like_superlike: like_get_urlpic error Error: Protocol error (Page.navigate): Invalid parameters url: string value expected
[INFO] like_superlike: try open userpage
[DEBUG] like_superlike: Error: waiting for selector "main article:nth-child(1) header:nth-child(1) div:nth-child(2) a:nth-child(1)" failed: timeout 30000ms exceeded
[INFO] like_superlike: photo not found, retry next...
[INFO] like_superlike: like_get_urlpic
[DEBUG] like_superlike: array photos 
[INFO] like_superlike: current photo url undefined
[WARNING] like_superlike: check if current hashtag have photos, you write it good in config.js? Bot go to next hashtag.
[ERROR] like_superlike: like_get_urlpic error Error: Protocol error (Page.navigate): Invalid parameters url: string value expected
[INFO] like_superlike: try open userpage
[DEBUG] like_superlike: Error: waiting for selector "main article:nth-child(1) header:nth-child(1) div:nth-child(2) a:nth-child(1)" failed: timeout 30000ms exceeded
[INFO] like_superlike: photo not found, retry next...
[INFO] like_superlike: like_get_urlpic
[DEBUG] like_superlike: array photos 
[INFO] like_superlike: current photo url undefined
[WARNING] like_superlike: check if current hashtag have photos, you write it good in config.js? Bot go to next hashtag.
[ERROR] like_superlike: like_get_urlpic error Error: Protocol error (Page.navigate): Invalid parameters url: string value expected
[INFO] like_superlike: try open userpage

```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. cd Desktop/instabotjs
2. node bot.js
3. wait some like

### config.js (remember to remove password)
```
module.exports = {
    // InstagramBot.js Configs
    "debug": true,
    "login": true,
    "ui": false, // only for social-manager-tools

    // Instagram Account
    "instagram_username": "", // without @
    "instagram_password": "",
    "instagram_hashtag": ["travel", "travelgram", "vlogger", "videomaker", "photographer", "supercar", "Supercars", "ferrari", "porsche", "youtube", "adventure", "wanderlust", "vacation", "explore", "traveler", "travelphotography"], //without #, raccomanded more than 20
    "instagram_pin": "sms", //method to receive pin (email or sms)

    // BOT Configs
    // [WORK] likemode_classic          - select random hashtag from config list and like 1 random photo (of last 20) | 400-600 like/day.
    // [WORK] likemode_realistic        - select random hashtag from config list, like fast 10-12 photo and sleep 15-20min. Sleep at night
    // [WORK] likemode_superlike        - select random hashtag from config list and like 3 random photo of same user | 400-600 like/day.
    // [WORK] likemode_competitor_users - select account, select random followers, like 10-12 photo and sleep 15-20min.
    // [WORK] comment_mode              - leave a comment under the photo via hashtag.
    // [WORK] fdfmode_classic           - follow 30 users, and defollow the first followed at 31 follow (in loop). This method is not detected from socialblade. ~1h | 300/300 follow-defollow/day.
    "bot_mode": "likemode_realistic",
    "bot_likeday_min": 800,      // work only in likemode_classic
    "bot_likeday_max": 900,      // work only in likemode_classic. In 2018 limit instagram is 1000/day
    "bot_sleep_night": false,    // if you need stop bot in night 
    "bot_start_sleep": "7:00",   // sleep from 00:00 to 7:00 am, work in likemode_realistic, comment_mode, likemode_superlike and likemode_competitor_users
    "bot_superlike_n": 3,        // superlike number of likes for user
    "bot_followday": 300,        // number of follow-defollow user/day, work in fdfmode_classic. In 2018 limit instagram is 500/day
    "bot_followrotate": 30,      // follow 30 users, and defollow the first followed at 31 follow (in loop), work in fdfmode_classic | If is 0 defollow all followed previously by bot
    "bot_userwhitelist": ["giulyegiammy", "powerslidelover", "ste_tami"],   // usernames without @ | don't defollow this users on follow-defollow mode, work in fdfmode_classic
    "comment_mode": {
        "comments": {
            "type": "array",
            "source": ["comment1", "comment2", "comment3"],
        }
    },
    "likemode_competitor_users": {
        "account": "user_nickname" // without @
    },

    // Puppeteer Configs
    "chrome_headless": true,
    "chrome_options": ["--disable-gpu", "--no-sandbox", "--window-size=1920x1080"],
    "executable_path": "/usr/bin/chromium-browser", // example for Mac OS: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome

    // DATABASES
    "fdfdatabase_path": "./databases/fdf.db", // databases with username of followed users with follodefollow mode
    "logdb_path": "./databases/logs.db",      // logs in sqllite database for better report

    // LOG
    "pin_path": "./loginpin.txt",
    "log_path": "./logs/debug.log",
    "logerr_path":"./logs/errors.log",
    "screenshot_path": "./logs/screenshot/",
    "log": {
        "drivers": ["console"], // slack or console
        "screenshot": false,  // disable or enable screenshot in logs folder
        "channels": {
            "console": "",
            "slack": {
                "webhook": ""
            }
        }
    }
};


```

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot. 

![image](https://user-images.githubusercontent.com/19846692/45499785-63535200-b77d-11e8-9d32-c015bef73d0e.png)

**Comment by ptkdev** [comment]
Imho is duplicate than #84

**Comment by focamonca** [comment]
> Imho is duplicate than #84

well this is another bot_mode and a different error

**Comment by ptkdev** [comment]
Today instagram update we site = 100 new bug :) happening

**Comment by focamonca** [comment]
So, sorry for the flame

**Comment by ptkdev** [comment]
Fixed with **`v0.9.1`** release

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>  <a href="http://coffee.ptkdev.io" alt="Ko-fi Donate"><img src="https://img.shields.io/badge/buy%20me-coffee-4B788C.svg"></a>
